rules/linux/credential_access_unusual_instance_metadata_service_api_request.toml

```diff
@@ -0,0 +1,83 @@
+[metadata]
+creation_date = "2024/08/22"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/08/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies potentially malicious processes attempting to access the cloud service provider's instance metadata service (IMDS) API
+endpoint, which is typically used to retrieve sensitive instance-specific information such as instance ID, public IP
+address, and even temporary security credentials if role's are assumed by that instance. The rule monitors for various tools and scripts like curl, wget,
+python, and perl that might be used to interact with the metadata API.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Unusual Instance Metadata Service (IMDS) API Request"
+references = ["https://hackingthe.cloud/aws/general-knowledge/intro_metadata_service/"]
+risk_score = 47
+rule_id = "ecc0cd54-608e-11ef-ab6d-f661ea17fbce"
+severity = "medium"
+tags = [
+    "Domain: Endpoint",
+    "OS: Linux",
+    "Use Case: Threat Detection",
+    "Tactic: Credential Access",
+    "Tactic: Discovery",
```

Comment:
There's discovery here, but I don't see it in the MITRE listing

Reply:
Added